guides/scripts/extract_links_toc

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/ruby
+
+# This is a utility to extract the structure of the documentation as a TOC-like json file
+# this json file will be used later to verify links to the documentation in the Foreman UI.
+
+require 'nokogiri'
+require 'json'
+
+unless (f2l_file = ENV['F2L'])
+  STDERR.puts "Must specify folder to link file: F2L="
+  return 1
```

Comment:
Does `return` work? Should it be `exit`?
```suggestion
  exit 1
```

Reply:
Yup, it works correctly:

```
[root@6125b009b90d guides]# ruby scripts/extract_links_toc build/**/index-*.html > build/toc.json
scripts/extract_links_toc: warning: argument of top-level return is ignored
Must specify folder to link file: F2L=
[root@6125b009b90d guides]# 

```

Reply:
It doesn't:
```console
$ ./guides/scripts/extract_links_toc 
./guides/scripts/extract_links_toc: warning: argument of top-level return is ignored
Must specify folder to link file: F2L=
$ echo $?
0
```

And it tells you that it ignores the return value.